config/application.rb

```diff
@@ -46,5 +46,9 @@ class Application < Rails::Application
     end
 
     config.exceptions_app = routes
+
+    config.cache_store = :redis_store, { host: 'localhost',
+                                         port: '6379',
+                                         namespace: 'cache' }
```

Comment:
Co do namespace to wątpie, że ktoś kiedyś odpali różne środowiska na jednej maszynie ale mogę dodać tego env'a. Co do osobnego pliku to zastanawiam się czy nie wyrzucić `host` i `port` - są to wartości domyślne, których raczej nie będziemy zmieniać. Wtedy nie ma sensu aby wyrzucać to do osobnego pliku. (Jeszcze o tym pomyślę)